authoidc.properties

```diff
@@ -1,12 +1,18 @@
 category=Integration
 description=Delegate authentication using the OpenID Connect protocol
 homepageUrl=https://github.com/vaulttec/sonar-auth-oidc
-archivedVersions=1.0.4,1.1.0,2.0.0
-publicVersions=2.1.0
+archivedVersions=1.0.4,1.1.0,2.0.0,2.1.0
+publicVersions=2.1.1
 
 defaults.mavenGroupId=org.vaulttec.sonarqube.auth.oidc
 defaults.mavenArtifactId=sonar-auth-oidc-plugin
 
+2.1.1.description=Fixes some issues with the newly added auto-login servlet filter
+2.1.1.sqVersions=[7.9,LATEST]
```

Comment:
This range needs to start with 8.9. 
7.9 is fully past EOL and sunsetted now.

Reply:
This plugin is [compiled and tested against SQ 7.4](https://github.com/vaulttec/sonar-auth-oidc/blob/master/pom.xml#L26). But as requested I'll limit this plugin's marketplace entry to SQ 8.9 and newer.